src/library/librarycontrol.cpp

```diff
@@ -655,8 +655,8 @@ void LibraryControl::slotSortColumn(double v) {
 
 void LibraryControl::slotSortColumnToggle(double v) {
     int column = static_cast<int>(v);
-    if (static_cast<int>(m_pSortColumn->get()) == column) {
-        m_pSortOrder->set(!m_pSortOrder->get());
+    if (column == static_cast<int>(m_pSortColumn->get())) {
+        m_pSortOrder->set((m_pSortOrder->get() == 0) ? 1 : 0);
```

Comment:
Now you set a double with an int instead of a boolean.
Was the compiler complaining? 
Is it satisfied now?
I think it should be 1.0 and 0.0.

Reply:
No, the compiler doesn't complain. It didn't complain because we were setting a double with a bool btw, it complained because we were implicitly casting a double to bool with the `!` operator. Casting an int to bool doesn't lead to a precision loss, so I guess it's fine. I'll update the literals anyway.